### PR TITLE
New profile page

### PR DESCRIPTION
**Pull request by zerefwayne**
image

image

### CODE REVIEW

File: src/app/profile/user/details/details.component.html [hunk 1]
```diff
@@ -1,223 +1,89 @@
...
-                    <div class="block" *ngIf="user?.technologies !== null || user?.technologies?.length !== 0">
-                      <h4 class="title">I am pretty good at</h4>
-                      <div class="tech-stack">
-                        <div class="tech-stack__item" *ngFor="let tech of user.technologies">{{ tech }}</div>
-                      </div>
-                    </div>
+    <div class="profile-header-info">
+      <div class="container">
+        <div class="profile-header-info__user-card">
+          <div class="profile-header-info__photo"><img src="../../../../assets/user.png" /></div>
```
[the remark this answers is not in the document]
Comment (WickedBrat) [reply src/app/profile/user/details/details.component.html]: `<cl-image class="dp-l-image" public-id="{{ user.profile_pic }}" secure="true"></cl-image>`

File: src/app/profile/user/details/details.component.html [hunk 2]
```diff
@@ -1,223 +1,89 @@
...
-                <ng-template matTabContent>
-                  <div class="tab-contact">
-                    <div class="profile-details">
-                      <div class="detail-block">
-                        <mat-icon class="detail-block__icon">mail_outline</mat-icon> {{ user.email }}
-                      </div>
-                      <div class="detail-block">
-                        <mat-icon class="detail-block__icon">location_on</mat-icon> Lucknow, India
-                      </div>
+          <h3 class="profile-header-info__username">@aayushjoglekar</h3>
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.html]: `user.username`

File: src/app/profile/user/details/details.component.html [hunk 3]
```diff
@@ -1,223 +1,89 @@
...
+        </div>
 
-                      <div class="detail-block">
-                        <mat-icon class="detail-block__icon">cake</mat-icon> {{ user.birth_date | date: 'longDate' }}
-                      </div>
+        <div class="profile-header-info__name">Aayush Joglekar</div>
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.html]: `{{ user.name }}`

File: src/app/profile/user/details/details.component.html [hunk 4]
```diff
@@ -1,223 +1,89 @@
...
-                  </div>
-                </ng-template>
-              </mat-tab>
-            </mat-tab-group>
+  <div class="main-body">
+    <div class="container">
+      <div class="side-info">
+        <div class="info_card">
+          <div class="info_card--header">Location</div>
+          <div class="info_card--body">IIIT Allahabad, India</div>
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.html]: `{{ user.location }}`

File: src/app/profile/user/details/details.component.html [hunk 5]
```diff
@@ -1,223 +1,89 @@
...
+        </div>
+        <div class="info_card">
+          <div class="info_card--header">Bio</div>
+          <div class="info_card--body">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae esse eos officiis, nesciunt expedita deleniti
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.html]: `{{ user.bio }}`

File: src/app/profile/user/details/details.component.html [hunk 6]
```diff
@@ -1,223 +1,89 @@
...
+            explicabo quo! Nulla commodi reprehenderit recusandae ex cupiditate id eos porro repellat enim, maxime
+            omnis!
           </div>
         </div>
+        <div class="info_card">
+          <div class="info_card--header">Tech Stack</div>
+          <div class="info_card--body">React, Node, Bootstrap</div>
```
Comment (WickedBrat) [inline src/app/profile/user/details/details.component.html]:
```javascript
	<span *ngFor="let tech of user.technologies"> {{ tech }} </span>
```

File: src/app/profile/user/details/details.component.html [hunk 7]
```diff
@@ -1,223 +1,89 @@
...
+        <div class="main-info__projects">
+          <div class="project_card">
+            <div class="project_card--header">Project Name | Ideation</div>
+            <div class="project_card--body">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident qui saepe nam, dicta omnis quidem
+              velit voluptatem tempore? Repudiandae, id.
             </div>
-            <div class="detail-block" [ngStyle]="{ display: !isFieldEditable ? 'block' : 'none' }">
-              <mat-icon class="detail-block__icon">cake</mat-icon> {{ user.date_of_birth | date: 'longDate' }}
+            <a class="expand" href="#"><img src="../../../../assets/images/external-link.svg"/></a>
```
[the remark this answers is not in the document]
Comment (WickedBrat) [reply src/app/profile/user/details/details.component.html]: use `<mat-icon>open_in_new</mat-icon>` and likewise below

File: src/assets/images/external-link.svg [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (WickedBrat) [inline src/assets/images/external-link.svg]: remove this.

**Comment by zerefwayne** [comment]
@WickedBrat I did the changes. Please have a look.

**Pull request merged**